php/default-filters.php

```diff
@@ -6,6 +6,10 @@
  *
  * Not all of the default hooks are here (yet).
  *
+ * @since 0.11.3-beta1 Some hooks in this file may change to support new
```

Comment:
I would make this a bit louder and more alarming. s/may/will/ and don't be afraid to use some caps, like "HEADS UP!" or "IMPORTANT!"